### ISSUE TITLE
Can't serialize a type with a field named 'key'

### ISSUE THREAD

**Issue by dhasenan** [rcorre/jsonizer#36]
Tested with 0.7.0. (I encountered other issues with 0.7.1 that I'm still needing to track down.)

```D
class A {
  mixin JsonizeMe;
  @jsonize:
  int key;
  int other;
}
```

Error produced:

    ../../../.dub/packages/jsonizer-0.7.0/src/jsonizer/jsonize.d-mixin-80(80,11): Error: cannot implicitly convert expression (val) of type string to int

Move the `@jsonize:` line down one and the error disappears.

**Comment by rcorre** [comment]
That should be fixed in v0.7.1. I just found another significant bug in v0.7.1 regarding the use of bare `@jsonize`. Give v0.7.2 a try -- I just pushed it up.

**Comment by michael-fadely** [comment]
Similar deal with fields called 'name', except that it fails a little more spectacularly.

**Comment by rcorre** [comment]
Good catch @SonicFreak94! Should be fixed in `v0.7.4`.
